script/github-actions/codecoverage-format-report.js

```diff
@@ -0,0 +1,40 @@
+const fs = require('fs');
+const path = require('path');
+
+const codecoverageReport = fs.readFileSync(
+  path.join(__dirname, '../../test-results/coverage_report.txt'),
+  'utf8',
+);
+
+// eliminiate unnecessary lines & spaces from coverage report
+const regex = /([^A-Za-z0-9\/()%-])+\s/g; // eslint-disable-line no-useless-escape
+
+const codecoverageData = codecoverageReport.replace(regex, ',').split(',');
+
+let codecoverageMarkdown = '<table> \n <thead> \n <tr> \n'; // markdown format
```

Comment:
This is technically HTML, which is accepted in Markdown, so this works!

For your awareness, there is also some [Markdown-specific syntax that's a more shorthand way of forming tables](https://www.markdownguide.org/extended-syntax/#tables).

Reply:
This is how I originally had it previously. Unfortunately, it kept intepretting it as a string and not so much markdown anymore.

https://github.com/department-of-veterans-affairs/vets-website/runs/2412176374?check_suite_focus=true -- test run that I did

Reply:
Changed variable name to `codeCoverageHTML`